### ISSUE TITLE
An error occurred while Style/RaiseArgs cop was inspecting...

### ISSUE THREAD

**Issue by samphilipd** [bbatsov/rubocop#3255]
I get this error in a few of my controllers when using auto-correct:

`An error occurred while Style/RaiseArgs cop was inspecting test.rb`

--------

## Expected behavior

Should auto-correct the file.

## Actual behavior

Prints error.

## Steps to reproduce the problem

Run `rubocop --auto-correct` on the following file:

```ruby
raise AccessDenied.new unless foo
```

## RuboCop version

```
$ rubocop -V
0.41.1 (using Parser 2.3.1.2, running on ruby 2.3.1 x86_64-darwin15)
```
